core/domain/topic_services_test.py

```diff
@@ -835,23 +858,6 @@ def test_update_topic_language_code(self):
         topic = topic_services.get_topic_by_id(self.TOPIC_ID)
         self.assertEqual(topic.language_code, 'bn')
 
-    def test_update_topic_additional_story_ids(self):
```

Comment:
Shouldn't this test still exist?

Reply:
Not this, but yeah, added the `add_additional_story` and `delete_additional_story` tests.